packages/docs/docs/access/access-policies.md

```diff
@@ -1,9 +1,9 @@
 ---
-sidebar_position: 4
+sidebar_position: 0
 tags: [auth]
 ---
 
-# Access Controls
```

Comment:
One thing here that I'd like to preserve.  Access Controls is a widely known industry standard term that people understand... while "access policies" unfortunately is not.  I'd like to maintain Access Controls as the top level name for clarity while heavily referring to "Access Policies" as our implementation.

Reply:
I'm going to push back on this. I maintained the term "Access Control" at the top level to maintain the industry standard term. Below, I tried to list the different _tools_ we have for to implement access control, of which `AccessPolicies` are one. 


<img width="294" alt="image" src="https://github.com/medplum/medplum/assets/1259236/48763980-46a2-4f57-97d5-3d3780b442ca">

Additionally, a lot of our devs use the search term "Access Policy" on our docs

<img width="1104" alt="image" src="https://github.com/medplum/medplum/assets/1259236/5beb11e0-977d-4d10-9824-21969fc6c5a4">